app/services/sources/TwitterSource.java

```diff
@@ -229,4 +229,8 @@ private void setPostURL(Status s, Post.Builder builder) {
 		}
 	}
 
+	@Override
+	public String getSourceName() {
+		return TWITTER;
```

Comment:
source name used for data tier should fall under source namespace, e.g. `"source:twitter"`. Use static fields `SOURCE_NAMESPACE` and `NAMESPACE_DELIMITER` in AbstractDataAccess. Can either change here, or in call to `dataAccess.addNewPosts(...)` in `AbstractDataCollector`.